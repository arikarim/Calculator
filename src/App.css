body {
  overflow-x: hidden;
}

.button1 {
  width: 60px;
}
