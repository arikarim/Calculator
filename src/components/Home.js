import React from 'react';

const Home = () => (
  <div>
    <h1>Math Magicians</h1>
    <h6>Welcome to our page!</h6>
  </div>
);

Home.propTypes = {

};

export default Home;
