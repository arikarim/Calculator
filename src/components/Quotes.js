import React from 'react';

const Quotes = () => (
  <div>
    <p className="title">
      Mathematics is not about numbers, equations, computations, or
      algorithms: it is about understanding. â€“William Paul Thurston
    </p>
  </div>
);

Quotes.propTypes = {

};

export default Quotes;
