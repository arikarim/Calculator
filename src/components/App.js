import '../App.css';
import React from 'react';
import Display from './Display';
import ButtonPanel from './ButtonPanel';

const App = () => (
  <div className="">
    <div className="col-2 mx-auto">
      <div className="text-end my-2">
        <Display />
      </div>
      <ButtonPanel />
    </div>
  </div>
);

App.propTypes = {

};

export default App;
