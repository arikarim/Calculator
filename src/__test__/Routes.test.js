import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import App from '../components/App';
import Home from '../components/Home';
import Quotes from '../components/Quotes';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <>
        <BrowserRouter>
          <Switch>
            <Route path="/" exact component={Home} />
            <Route path="/Calculator" component={App} />
            <Route path="/Quotes" component={Quotes} />
          </Switch>
        </BrowserRouter>
        <h1>Math Magicians</h1>
        <h6 className="title">Welcome to our page!</h6>
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
