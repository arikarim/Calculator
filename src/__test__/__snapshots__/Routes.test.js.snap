// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div>
    <h6
      className="title"
    >
      Welcome to our page!
    </h6>
  </div>,
  <h1>
    Math Magicians
  </h1>,
  <h6
    className="title"
  >
    Welcome to our page!
  </h6>,
]
`;
